components/06-examples/aanvragen.hbs

```diff
@@ -16,31 +17,14 @@
         {{/each}}
       {{/accordion}}
     </div>
-
-    <div class="col-md-3">
-      {{#progressBlock }}
-        <h2>Aanvraag voorbereiden</h2>
-        <p>Omgevingsvergunning</p>
-        <ul class="list-unstyled">
-          <li class="active">
-            {{> '@icon' icon="fas fa-angle-right"}} Omschrijving project {{> '@icon' icon="fas fa-check"}}
-          </li>
-          <li class="active">
-            {{> '@icon' icon="fas fa-angle-right"}} Aanvrager {{> '@icon' icon="fas fa-check"}}
-          </li>
-          <li>
-            {{> '@icon' icon="fas fa-angle-right"}} Locatie
-          </li>
-          <li>
-            {{> '@icon' icon="fas fa-angle-right"}} Werkzaamheden
-          </li>
-          <li>
-            {{> '@icon' icon="fas fa-angle-right"}} Bijlagen
-          </li>
-        </ul>
-      {{/progressBlock}}
-    </div>
+       
   </div>
+    <div class="row">
+        <div class="col-md-3 col-md-push-3 col-md-offset-6">{{> '@button' label='Volgende stap' type='button' modifier='primary pull-right'}}
+          </div>
+          <div class="col-md-3 col-md-pull-9">{{> '@button' label='Vorige stap' type='button' modifier='default'}}
+          </div>
+    </div>
 </main>
-
+    
```

Comment:
Als je de file dan toch open hebt, deze spaces zijn niet nodig

Reply:
Space is verwijderd.